src/main/java/io/cdap/e2e/pages/actions/CdfConnectionActions.java

```diff
@@ -0,0 +1,204 @@
+    SeleniumHelper.getPropertiesLocators(CdfConnectionLocators.class);
+  }
+
+  /**
+   * Open Wrangler connections page
+   */
+  public static void openWranglerConnectionsPage() throws IOException {
+    SeleniumDriver.openPage(SeleniumHelper.readParameters(ConstantsUtil.WRANGLER_CONNECTIONS_URL));
+    PageHelper.acceptAlertIfPresent();
+    WaitHelper.waitForPageToLoad();
+  }
+
+  /**
+   * Click on the Browser connections button inside Plugin's properties page
+   */
+  public static void clickBrowseConnectionsButton() {
+    ElementHelper.clickOnElement(CdfConnectionLocators.browseConnectionsButton);
+  }
+
+  /**
+   * Select the connection
+   *
+   * @param connectionName If connectionName is present in {@link ConstantsUtil#DEFAULT_PLUGIN_PROPERTIES_FILE} as key
+   *                       then actual connection to select is fetched from it
+   *                       else connectionName is selected as it is.
+   */
+  public static void selectConnection(String connectionName) {
+    WebElement connectionRowElement;
+
+    String connectionNameFromPluginPropertiesFile = PluginPropertyUtils.pluginProp(connectionName);
+    if (connectionNameFromPluginPropertiesFile == null) {
+      connectionRowElement = CdfPluginPropertiesLocators.locateElementWithText(connectionName);
+    } else {
+      connectionRowElement = CdfPluginPropertiesLocators.locateElementWithText(connectionNameFromPluginPropertiesFile);
+    }
+
+    ElementHelper.clickOnElement(connectionRowElement);
+    WaitHelper.waitForElementToBeDisplayed(connectionRowElement);
+  }
+
+  /**
+   * Verify the test connection is successful
+   */
+  public static void verifyTheTestConnectionIsSuccessful() {
+    WaitHelper.waitForElementToBePresent(CdfPluginPropertiesLocators
+                                           .locatorOfElementWithText("Successfully connected."));
+    AssertionHelper.verifyElementDisplayed(CdfPluginPropertiesLocators
+                                             .locateElementWithText("Successfully connected."));
+  }
+
+  /**
+   * Verify the connection is created
+   *
+   * @param connectionName If connectionName is present in {@link ConstantsUtil#DEFAULT_PLUGIN_PROPERTIES_FILE} as key
+   *                       then actual connectionName is fetched from it
+   *                       else connectionName param used as it is.
+   */
+  public static void verifyConnectionIsCreatedSuccessfully(String connectionName) {
+    String connectionNameFromPluginPropertiesFile = PluginPropertyUtils.pluginProp(connectionName);
+    if (connectionNameFromPluginPropertiesFile == null) {
+      WaitHelper.waitForElementToBeDisplayed(CdfPluginPropertiesLocators.locateElementWithText(connectionName));
+      return;
+    }
+    WaitHelper.waitForElementToBeDisplayed(CdfPluginPropertiesLocators
+                                             .locateElementWithText(connectionNameFromPluginPropertiesFile));
+  }
+
+  /**
+   * Select data row for connection
```

Comment:
What do you mean by data row, could you please mention some examples in comments?

Reply:
Added example in comments.